bin/commander

```diff
@@ -8,30 +8,78 @@ program :version, Commander::VERSION
 program :description, 'Commander utility program.'
 
 command :init do |c|
-  c.syntax = 'commander init <file>'
+  c.syntax = 'commander init [option] <file>'
```

Comment:
Thanks for that catch. Makes sense.
On Fri, 2016-02-19 at 11:09 -0800, Gabriel Gilder wrote:
> In bin/commander:
> > @@ -8,30 +8,78 @@ program :version, Commander::VERSION
> >  program :description, 'Commander utility program.'
> >  
> >  command :init do |c|
> > -  c.syntax = 'commander init <file>'
> > +  c.syntax = 'commander init [option] <file>'
> Instead of [option] I would just put [--modular] since it's the only
> option.
> —
> Reply to this email directly or view it on GitHub.
-- 
> It is as easy to create a castle as a button. It's just a matter of whether you're focused on a castle or a button.
> 
> --- Abraham